frontend/resourceadm/components/ResourceNameAndId/ResourceNameAndId.tsx

```diff
@@ -174,12 +188,14 @@ export const ResourceNameAndId = ({
           </div>
         </div>
       </>
-    )
-  }
+    );
+  };
 
   return (
     <>
-      <Paragraph size='small'>{text}</Paragraph>
+      <Paragraph size='small' className={spacingTop && classes.spaceTop}>
```

Comment:
For å separere stil og funksjonell kode så mye som mulig, vil vi unngå props med navn som "spacingTop". Hvis man tenker på hvordan komponenten ser ut på de stedene i koden hvor den brukes, ville det beste vært å bare ha en `className`-prop som setter en klasse på en wrapper rundt hele komponenten. Er det noen grunn til å ikke ha en slik wrapper her?